packages/server/test/unit/socket_spec.js

```diff
@@ -783,6 +783,40 @@ describe('lib/socket', () => {
         })
       })
     })
+
+    context('on(get:cached:test:state)', () => {
```

Comment:
More of a nit but not a big deal since `context` isn't available in vitest
```suggestion
    describe('on(get:cached:test:state)', () => {
```